ci/test_wheel.sh

```diff
@@ -7,20 +7,26 @@ mkdir -p ./dist
 RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
 RAPIDS_PY_WHEEL_NAME="cuml_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 python ./dist
 RAPIDS_PY_WHEEL_NAME="libcuml_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 cpp ./dist
+RAPIDS_TESTS_DIR=${RAPIDS_TESTS_DIR:-"${PWD}/test-results"}
+mkdir -p "${RAPIDS_TESTS_DIR}"
 
-# echo to expand wildcard before adding `[extra]` requires for pip
+# Install just minimal dependencies first
 rapids-pip-retry install \
   ./dist/libcuml*.whl \
-  "$(echo ./dist/cuml*.whl)[test]"
+  ./dist/cuml*.whl
 
-RAPIDS_TESTS_DIR=${RAPIDS_TESTS_DIR:-"${PWD}/test-results"}
-mkdir -p "${RAPIDS_TESTS_DIR}"
+# Try to import cuml with just a minimal install"
+rapids-logger "Importing cuml with minimal dependencies"
+python -c "import cuml"
+
+# echo to expand wildcard before adding `[extra]` requires for pip
+rapids-pip-retry install \
+  "$(echo ./dist/cuml*.whl)[test]"
```

Comment:
```suggestion
   ./dist/libcuml*.whl \
  "$(echo ./dist/cuml*.whl)[test]"
```

I think you should repeat the install of the locally-download `libcuml`. Otherwise, `pip` may choose to replace the existing installation with a newer one from the nightly index or `pypi.nvidia.com`.

Passing a list of wheel files via requirements gets you a strong guarantee that either all of those wheels will be installed or you get a big loud error explaining why they couldn't be.